app/models/financialDetails/Payment.scala

```diff
@@ -30,7 +30,8 @@ case class Payment(reference: Option[String],
                    dueDate: Option[LocalDate],
                    documentDate: LocalDate,
                    transactionId: String,
-                   mainType: Option[String] = None)
+                   mainType: Option[String] = None,
```

Comment:
can we check if we can remove defaults for main and mainTransactions ?

Reply:
Removed